syntax = "proto3";
package vire.cmslapp;

import "vire/utility/BasePayload.proto";
import "vire/utility/InstanceIdentifier.proto";

message ConnectionRequest {

  vire.utility.BasePayload _base = 1; // Base class

  vire.utility.InstanceIdentifier setup_id            = 100; // The setup identifier
  repeated string                 requested_resources = 101; // The paths of the requested resources

}
